server/item.go

```diff
@@ -50,6 +51,10 @@ func (s *RESTServer) SlotHandler(w http.ResponseWriter, r *http.Request, ps http
 		} else {
 			w.WriteHeader(404)
 		}
+		// We were unaware of recall errors until it was reported upstream
+		// this appears to capture those issues. However, we may need further
+		// logic to not send every error.
+		raven.CaptureError(err, nil)
```

Comment:
This will run for every 404 and every tape-in-use request. probably want to be more discerning. maybe need to test for a NotFound error explicitly.

Reply:
I'm at a loss to navigate this. I don't think my PR is reporting the exception at the right place. I think there is a lower level place that this reporting would make more sense.

I have looked at the code, and today it is beyond me.

Reply:
Your intuition is spot on. There should be a call to sentry at each unhandled error, which is usually where they are logged. This log has both handled and unhandled errors. I’ll take a look later today.

Reply:
And now on rereading your code I notice that you already put my concern in the comment. I was too tired last night to notice it! 

I reworked your idea. if you like it, please squash it back with yours, take credit for it, and force push the branch back. Then I'll merge.